src/DataBuilder.php

```diff
@@ -193,10 +201,12 @@ protected function setRequestParams($config)
 
     protected function setRequestBody($config)
     {
+        
         $this->requestBody = $this->tryGet($config, 'requestBody');
         
-        if (!$this->requestBody) {
+        if (!$this->requestBody && $this->includeRawRequestBody) {
             $this->requestBody = file_get_contents("php://input");
+            $_SERVER['php://input'] = $this->requestBody;
```

Comment:
@rokob For the same reasons I opt in for `include_raw_request_body` to be disabled by default.